androidApp/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
-    <string name="app_name">Press</string>
+  <string name="app_name">Press</string>
+
+  <item name="homeViewId" type="id" />
+  <item name="editorViewId" type="id" />
+
+  <!-- Unique integers to be used in EditorView to store some transient states -->
```

Comment:
I don't think this comment is needed. Anyone who's reading this file can check the usages where they're being used.

Reply:
Yeah. Sometime I feel like it is not how people are doing things out there....